users/templates/user_notifications_list.html

```diff
@@ -1,36 +1,56 @@
 {% extends 'base.html' %}
 
 {% load i18n %}
+{% load static %}
 {% load data_structure_element_selectors %}
 
+{% block extra_head %}
+    <link
+        rel="stylesheet"
+        href="https://code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css"
+        integrity="sha384-Nlo8b0yiGl7Dn+BgLn4mxhIIBU6We7aeeiulNCjHdUv/eKHx59s3anfSUjExbDxn"
+        crossorigin="anonymous"
+    />
+    <link
+        rel="stylesheet"
+        type="text/css"
+        href="{% static 'users/style.css' %}"
+    />
```

Comment:
@Karrp please, if you do something first see what have be done before. Please add  subresource integrity also here.

Reply:
Done